hardware/HarmonyHub.cpp

```diff
@@ -724,7 +717,7 @@ bool CHarmonyHub::SubmitCommand(const std::string &strCommand, const std::string
 		lstrCommand = GET_CURRENT_ACTIVITY_COMMAND;
 	}
 
-	std::string strData;
+	std::string strData = ""; // make sure to initialize this as we will be appending to it
```

Comment:
I think the constructor of this class will already do this

Reply:
Should not... it perfectly legal and normal, nobody would declare (i hope) things like

std::string myString("");